src/core/unit.h

```diff
@@ -421,6 +421,9 @@ typedef struct Unit {
         /* Is this a unit that is always running and cannot be stopped? */
         bool perpetual;
 
+        /* When true logs about this unit will be at debug level regardless of other log level settings */
+        bool debug_invocation;
```

Comment:
hmm, interesting that you made this a per-unit, rather than a per-service setting. What#s your rationale for that?

Reply:
Just because the logging happens with the unit object

Reply:
which logging specifcally?

Reply:
`log_unit*` are called with the unit object, and this is where this is checked. I don't mind moving it, but it means casting from unit to service. Up to you.

Reply:
so i think it might make sense to indeed leave this a property of the unit rather than the service, since I figure things like mount units might one day want something similar. who knows.